syft/workers/websocket_server.py

```diff
@@ -165,7 +166,11 @@ def start(self):
             )
 
         asyncio.get_event_loop().run_until_complete(start_server)
-        asyncio.get_event_loop().run_forever()
+        print("Serving. Press CTRL-C to stop.")
+        try:
+            asyncio.get_event_loop().run_forever()
+        except KeyboardInterrupt:
+            print("Server is ready.")
```

Comment:
The print statement should rather be: print("Websocket server stopped."). Because after hitting ctrl-c the websocket server won't respond any more.

Could we use logging instead of print statements?

Reply:
Yup, sure. Will make a commit to change the print statement into logging instead.

Reply:
Done.